### DISCUSSION TITLE
Storage service is always restaring while writing

### DISCUSSION THREAD

**Discussion by dpakpdl** [vesoft-inc/nebula#5162] [Q&A]
I have run Nebula Graph in docker in single machine, with 3 graphd, 3 metad and 3 storaged. When inserting a node, storaged cluster restarts and error is thrown.

`Error: RPC failure in StorageClient: Failed to write to remote endpoint. Wrote 0 bytes. AsyncSocketException: AsyncSocketException: connect failed, type = Socket not open, errno = 111 (Connection refused)` 

I am not able anything.

**Comment by QingZ11** [comment]
Please confirm whether storage / graph / meta services is healthy
